Week1/Week1 Solution/palindrom.py

```diff
@@ -0,0 +1,13 @@
+#Input the Any Number to check palindrome or not !
+n=int(input("Enter Any Number:"))

```

Comment:
According to definition you first need to check whether the entered value is integer or not..? 
So where is that statement...?